ngx-fudis/projects/ngx-fudis/src/lib/components/form/checkbox/checkbox.component.scss

```diff
@@ -69,6 +87,10 @@
     }
   }
 
+  &__required {
+    margin-left: spacing.$spacing-xs;
```

Comment:
Since there are no proper designs from this in Figma, I think this should be discussed with designers.

Primarly my conserns are the spacing and the font size and color, since they differ from other required texts we have in out components.

<img width="391" alt="Screenshot 2025-07-07 at 13 58 32" src="https://github.com/user-attachments/assets/928e9ce6-3335-4b05-afe9-3e442d2806bd" />